sdk/src/Services/DynamoDBv2/Custom/DocumentModel/Table.cs

```diff
@@ -841,18 +898,23 @@ public static bool TryLoadTable(IAmazonDynamoDB ddbClient,
         /// </returns>
         public static bool TryLoadTable(IAmazonDynamoDB ddbClient, TableConfig config, out Table table)
         {
-            if (config == null)
-                throw new ArgumentNullException("config");
-
-            try
-            {
-                table = LoadTable(ddbClient, config);
-                return true;
-            }
-            catch
+            var operationName = DynamoDBTelemetry.ExtractOperationName(nameof(Table), nameof(TryLoadTable));
+            var tracerProvider = ddbClient.Config.TelemetryProvider.TracerProvider;
+            using (DynamoDBTelemetry.CreateSpan(tracerProvider, operationName, spanKind: SpanKind.CLIENT))
             {
-                table = null;
-                return false;
+                if (config == null)
+                    throw new ArgumentNullException("config");
```

Comment:
nit: use `nameof(config)` instead of using string "config"

Reply:
This is existing code that was just placed inside the `using` scope. I fixed it